astropy/modeling/core.py

```diff
@@ -2416,24 +2417,82 @@ def render_model(model, arr=None, coords=None, copy=True):
     ----------
     model : `Model`
         Model to be evaluated.
-    arr : `numpy.ndarray`, optional
-        Array on which the model is evaluated.
+    arr : `numpy.ndarray` 
+        Array on which the model is evaluated. 
     coords : array-like, optional
-        Coordinate arrays mapping to ``array``, such that 
+        Coordinate arrays mapping to ``arr``, such that 
         ``arr[coords] == arr``.
+    wcs : `~astropy.wcs.WCS`, optional
+        A WCS object mapping to the input ``arr``. 
+    origin: int, optional
```

Comment:
The origin option can be removed.
Hard-code to use `0`.
There's some info why [here](https://gammapy.readthedocs.org/en/latest/development/index.html#pixel-coordinate-convention) and I can explain in the next hangout.
Basically – It's better to have a convention and use it consistently instead of letting the user configure this.